tests/unit/combo/rendering/rendering-test.js

```diff
@@ -478,7 +478,7 @@ QUnit.test('[ID7] igCombo scrolling tests', function (assert) {
 	assert.equal(combo.listScrollTop(), 200, "When value is undefined, then scroll top is returned.");
 	assert.equal($dropDownCont.prop('scrollTop'), 200, "Drop down container should be equal to set top scroll.");
 
-	scrollHeight = $dropDownCont.prop('scrollHeight') - $dropDownCont.innerHeight();
+	scrollHeight = Math.round( $dropDownCont.prop('scrollHeight') - $dropDownCont.innerHeight() + 0.188);
```

Comment:
Can't we avoid adding 0.188? Isnt this the aim of rounding - to avoid digits after the . ?

Reply:
Oh, of course we can. I forgot to remove it. Will remove it now :|